Formula/bareos-fd.rb

```diff
@@ -0,0 +1,71 @@
+class BareosFd < Formula
+  desc "Open-Source Data Protection Solution"
+  homepage "https://bareos.org"
+  url "https://github.com/bareos/bareos/archive/Release/17.2.4.tar.gz"
+  sha256 "4c443539012cf5ebb0fdb18878e604e82b951e6429c618acd18762f3c5724799"
+
+  depends_on "readline"
+  depends_on "openssl"
+
+  def install
+    system "./configure",
```

Comment:
I understand why you've done it but lord does this look weird 🙈.

Reply:
Agreed. Please use our usual style.